client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -73,6 +79,26 @@ const SiteTitle: React.FunctionComponent< Props > = ( { onSubmit, inputRef } ) =
 		_x( 'Max’s Burger Bar', 'sample site title' ),
 	];
 
+	React.useEffect( () => {
+		const fetchPodcastTitle = async () => {
+			if ( ! anchorFmPodcastId ) {
+				return;
+			}
+
+			let resp: Promise<object>;
+
+			try {
+				resp = await apiFetch( { path: `https://public-api.wordpress.com/wpcom/v2/podcast-details?url=https://anchor.fm/s/${ encodeURIComponent( anchorFmPodcastId ) }/podcast/rss` } );
```

Comment:
I'm using `apiFetch()` here because I was getting a JSON syntax error when trying to parse the response from `window.fetch()`. But if we stick with `apiFetch()` then I need to update the code to use its methods.